.github/workflows/test.yml

```diff
@@ -14,6 +14,20 @@ jobs:
   test:
     runs-on: ubuntu-latest
 
+    services:
+      mysql:
+        image: mysql:8.0
+        env:
+          MYSQL_ALLOW_EMPTY_PASSWORD: true
+          MYSQL_DATABASE: testing
+        ports:
+          - 3306/tcp
+        options: --health-cmd="mysqladmin ping" --health-interval=10s --health-timeout=5s --health-retries=3
+
+    env:
```

Comment:
Let's use the default values Laravel uses in their `.env.example file:

DB_DATABASE=laravel
DB_USERNAME=root
DB_PASSWORD=